projects/core/src/user/connectors/customer-coupon/customer-coupon.connector.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import {
+  CustomerCouponSearchResult,
+  CustomerCouponNotification,
+} from '../../../model/customer-coupon.model';
+import { CustomerCouponAdapter } from './customer-coupon.adapter';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CustomerCouponConnector {
+  constructor(protected adapter: CustomerCouponAdapter) {}
+
+  getMyCoupons(
+    userId: string,
+    pageSize: number,
+    currentPage: number,
+    sort: string
+  ): Observable<CustomerCouponSearchResult> {
+    return this.adapter.getMyCoupons(userId, pageSize, currentPage, sort);
```

Comment:
Why this method called "getMyCoupons"? Because all of the files and service methods use "CustomerCoupon". The same as adapter

Reply:
updated